scout/server/blueprints/variants/controllers.py

```diff
@@ -163,6 +165,80 @@ def parse_variant(store, institute_obj, case_obj, variant_obj, update=False):
     return variant_obj
 
 
+def variant_export_lines(store, case_obj, variants_query):
+    """Get variants info to be exported to file, one list (line) per variant"""
+
+    export_variants = []
+
+    for variant in variants_query:
+        variant_line = []
+        position = variant['position']
+        change = variant['reference']+'>'+variant['alternative']
```

Comment:
You do not need to change this! 
Just a suggestion for the future to use `join` when building strings.
This would then look like 
```python
'>'.join([variant['reference'],variant['alternative']])
```
🇵🇫